packages/nativescript-sdk/src/nativescript/init.ts

```diff
@@ -26,6 +27,7 @@ export function init(config?: KinveyConfig) {
     storageAdapter: getStorageAdapter(mergedConfig.storage),
     pubnub: PubNub
   })
+  live.startMonitoring();
```

Comment:
Should we start monitoring live service at init time, or only when we have an active user?

Reply:
Technically live service does not work without an active user but we have not overridden the `login()` function yet which we would need to in order to start monitoring the network. This is also slightly more complicated due to the many ways a user can login and the possibility that they also logout.

Reply:
As per our conversation, I think it would be best to tie the start/stop of monitoring to user login/logout, so we are not unnecessarily monitoring/initializing live service.